libminifi/include/ResourceClaim.h

```diff
@@ -55,23 +58,20 @@ class ResourceClaim : public std::enable_shared_from_this<ResourceClaim> {
 
   ResourceClaim(const std::string path, std::shared_ptr<core::StreamManager<ResourceClaim>> claim_manager, bool deleted = false);
   // Destructor
-  virtual ~ResourceClaim() {
+  ~ResourceClaim() {
```

Comment:
why remove the virtual?

Reply:
That class is not extended and there are no other virtual functions, thus we were creating a virtual table for this. There appears to be no intent to extend it so the performance impact seemed unnecessary.